api/helpers/gamestate/deal-cards.js

```diff
@@ -0,0 +1,80 @@
+const GameStatus = require('../../../utils/GameStatus.json');
```

Comment:
Like aliasing directories similar to what we have in vite? That could be convenient. We'd have to wrangle a separate config/setup from vite since this is backend, but it could be done